docs/topics/native/native-gradle.md

```diff
@@ -3,117 +3,131 @@
-  distributionType = 'BIN'
-}
-```
+   ```groovy
+   // build.gradle
+   plugins {
+       id 'org.jetbrains.kotlin.multiplatform' version '%kotlinVersion%'
+   }
+
+   repositories {
+       mavenCentral()
+   }
+
+   kotlin {
+     macosX64('native') { // on macOS
+     // linuxX64('native') // on Linux
+     // mingwX64('native') // on Windows
+       binaries {
+         executable()
+       }
+     }
+   }
+
+   wrapper {
+     gradleVersion = '%gradleVersion%'
+     distributionType = 'BIN'
+   }
+   ```
 
 </tab>
 </tabs>
 
-Next, create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+   You can use different [target presets](mpp-supported-platforms.md), such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
+   to define corresponding the target platform. The preset name describes a platform for which you are compiling your code.
+   These target presets optionally take the target name as a parameter, which is `native` in this case.
+   The target name is used to generate the source paths and task names in the project.
+
+3. Create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+
+4. Create a folder `src/nativeMain/kotlin` and place inside the `hello.kt` file with the following content:
+   
+   ```kotlin
+   fun main() {
+     println("Hello, Kotlin/Native!")
+   }
+   ```
 
-Depending on the target platform, different [functions](mpp-supported-platforms.md),
-such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
-are used for creating the Kotlin target. The function name is the platform for which you are compiling your code. 
-These functions optionally take the target name as a parameter, which is `"native"` in our case. 
-The specified _target name_ is used to generate the source paths and task names in the project.  
+   By convention, all sources are located in the `src/<target name>[Main|Test]/kotlin` folders, where `main` is for the source code
+   and `test` is for tests. `<target name>` corresponds to the target platform (in this case `native`), as specified in the build file.
 
-By convention, all sources are located in the `src/<target name>[Main|Test]/kotlin` folders, where `main` is for the source code
-and `test` is for tests. `<target name>` corresponds to the target platform (in this case `native`), as specified in the build file. 
+Now you are ready build your project and run the application. 
 
-Create a folder `src/nativeMain/kotlin` and inside it place the file `hello.kt` with the following contents:
+## Build and run the application
 
-```kotlin
-fun main() {
-  println("Hello Kotlin/Native!")
-}
-```
+1. From the root project folder, run the build command:
 
-## Build the project
+   ```bash
+   gradle nativeBinaries
+   ```
 
-From the root project folder, execute the build by running 
+   This command creates a folder `build/bin/native` and two subfolders inside: `debugExecutable` and `releaseExecutable`. They contain corresponding binary files.  
```

Comment:
"Folder" is a bit Windows-specific and belongs to the GUI concept; in the console-based tutorial the more general term `directory` would fit better.